CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# How to contribute
+
+## Open an issue
+
+Feel free to open an issue, No matter whether you found a bug, request a feature or have questions.
+
+1. Please check existing open and closed issues first and feel free to comment and re-open them.
+2. Use a meaningful title
+3. Provide enough context, a concrete use case and steps to reproduce bugs
+
+## Contributing code and documentation
+
+1. Create a fork of the repository
+2. Create a new branch holding your changes
+3. Apply your changes
+4. Commit your changes, follow the commit conventions: https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/en-us/Appendix/CommitMessage.html#commitmessage
```

Comment:
this guide is directly for gerrit, should we mention it? Not all of those conventions are relevant for github

Reply:
Fair point, will adjust. I set PR to draft until I've finished.